docs/source/transactions/transaction_protocol.rst

```diff
@@ -19,27 +19,31 @@ commit timestamp.
 Write Protocol
 --------------
 
-1. We buffer writes in memory until commit time. At commit time we grab
+1. Using the transactions start timestamp, complete all reads.  This is necessary to
```

Comment:
nit: sounds better as "Complete all reads using the transaction's start timestamp.". Should use an apostrophe. 

Not totally clear at this point why we're doing reads, but I think this is covered later.